#changeState is used to go through different states
int8 changeState
---
bool response
